packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -689,7 +690,9 @@ async function proxyRequestToAxios(
 			if (Buffer.isBuffer(responseData)) {
 				responseData = responseData.toString('utf-8');
 			}
-			error.message = `${response.status as number} - ${JSON.stringify(responseData)}`;
+			error.message = `${response.status as number} - ${JSON.stringify(
+				await binaryToBuffer(responseData),
+			)}`;
 		}
```

Comment:
Are we sure we want to do `binaryToBuffer(responseData)` here? We already stringify Buffer above and we should have a string ultimately. Am I missing something?

Reply:
we should either remove the lines 690-692, or replace it with `responseData = await binaryToBuffer(responseData)`.